### PR TITLE
Changes before publishing

### PR DESCRIPTION
**Pull request by ringods**
Hello @dominik-przybyl-wttech,

Sorry for the delay in reviewing this. 

I created this branch with a number of commits, each commit limited to a small change. I did this so you can easily follow the conceptual changes I made. The changes are mainly to the `Makefile` and to `provider.go`. All the other changed files in a commit are mainly due to regenerating the SDKs. Feel free to squash my commits in a single commit when you merge.

On top of the changes in this PR, I still want to ask you for a substantial change: renaming the repository from `pulumi-aem-native` to `pulumi-aem`. Within Pulumi, we have still a number of hardcoded assumptions that the name of the provider is the part after `pulumi-`. You named your provider `aem` ([code](https://github.com/wttech/pulumi-aem-native/blob/bc06d39bf6865e352ae88792649bd7ce0773ef65/provider/provider.go#L13)) but named the repository `pulumi-aem-native`. The fact this is a native provider is an implementation detail of no use to users of your provider, hence we suggest to drop the `-native` part from the repository name.

If you change the repository name, it has an impact on the Go module naming. So don't forget to update all references to `pulumi-aem-native` in the Go code to `pulumi-aem` and regenerate the Go client SDK.

### CODE REVIEW

**Review by krystian-panek-wttech — CHANGES_REQUESTED** [review]
@dominik-przybyl-wttech please repackage and change repo name as requested, thanks ;)

thank you @ringods for helping us

**Comment by dominik-przybyl-wttech** [comment]
done

File: provider/provider.go [hunk 1]
```diff
@@ -3,6 +3,11 @@ package provider
...
 	p "github.com/pulumi/pulumi-go-provider"
 	"github.com/pulumi/pulumi-go-provider/infer"
+	"github.com/pulumi/pulumi-go-provider/middleware/schema"
+	dotnetGen "github.com/pulumi/pulumi/pkg/v3/codegen/dotnet"
+	goGen "github.com/pulumi/pulumi/pkg/v3/codegen/go"
+	nodejsGen "github.com/pulumi/pulumi/pkg/v3/codegen/nodejs"
+	pythonGen "github.com/pulumi/pulumi/pkg/v3/codegen/python"
 	"github.com/pulumi/pulumi/sdk/v3/go/common/resource"
 	"github.com/pulumi/pulumi/sdk/v3/go/common/tokens"
 	"github.com/wttech/pulumi-aem-native/provider/instance"
```
Comment (krystian-panek-wttech) [inline provider/provider.go:13]: not repackaged?

Comment (dominik-przybyl-wttech) [reply provider/provider.go:13]: it is renamed in wttech/main brancj

**Pull request merged**